From: Alexander Mikhaylenko <alexander.mikhaylenko@puri.sm>
Date: Tue, 22 Sep 2020 15:22:42 +0500
Subject: librem5: adblock: Stop using g_task_set_name()

This is an addition in newer glib than we have.
---
 embed/ephy-filters-manager.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/embed/ephy-filters-manager.c b/embed/ephy-filters-manager.c
index b2c465f..b1bd728 100644
--- a/embed/ephy-filters-manager.c
+++ b/embed/ephy-filters-manager.c
@@ -268,14 +268,10 @@ filter_info_load_sidecar (FilterInfo          *self,
                               cancellable,
                               callback,
                               user_data);
-    g_autofree char *task_name = g_strconcat ("load sidecar file: ",
-                                              sidecar_file_path,
-                                              NULL);
     /* The FilterInfo itself as used async task data: it already contains
      * all the bits of information needed by the completion callback.
      */
     g_task_set_task_data (task, self, NULL);
-    g_task_set_name (task, task_name);
     g_file_load_bytes_async (sidecar_file,
                              g_task_get_cancellable (task),
                              (GAsyncReadyCallback)sidecar_bytes_loaded_cb,
@@ -326,11 +322,7 @@ filter_info_save_sidecar (FilterInfo          *self,
   g_autoptr (GBytes) data = filter_info_get_data_as_bytes (self);
   g_autoptr (GFile) sidecar_file = filter_info_get_sidecar_file (self);
   g_autofree char *sidecar_file_path = g_file_get_path (sidecar_file);
-  g_autofree char *task_name = g_strconcat ("save sidecar file: ",
-                                            sidecar_file_path,
-                                            NULL);
   GTask *task = g_task_new (NULL, cancellable, callback, user_data);
-  g_task_set_name (task, task_name);
 
   LOG ("Saving metadata: uri=<%s>, identifier=%s, checksum=%s, last_update=%" PRIu64,
        self->source_uri,
